lib/generate.js

```diff
@@ -128,8 +128,12 @@ function generateLayoutInternal(options, callback) {
             if (err) return callback(err);
             if (!image.width() || !image.height()) return callback(null, null);
 
-            // For the data layout JSON (when options.format is true), extract stretch metadata if present
-            if (options.stretchMetadata && options.format && img.svg.includes('mapbox-stretch')) {
+            // For the data layout JSON (when options.format is true), extract stretch and placeholder metadata if present
+            if (
+                options.stretchMetadata &&
```

Comment:
`options.stretchMetadata` is no longer totally precise for what this option enables, but I'm also hesitant to add more little flags (ie `options.placeholderMetadata`) to the `options` surface area when:

1. I'm not sure they would be used differently by any client, and
2. The downstream code is so similar